src/pages/SubmissionConfig.test.js

```diff
@@ -969,7 +971,7 @@ describe('Input source and requirements validation', () => {
 
     // ** ASSERT **
     // check that tooltip indicating missing required fields is present for Submit button
-    screen.getByText('One or more inputs have missing values')
+    screen.getByText('One or more inputs have missing/invalid values')
```

Comment:
Sorry to comment on tests instead of real code, but this is more of an "is this the outcome we want" style comment... Would it be hard to actually count the instances and make this be `"2 inputs have missing/invalid values"`